### DISCUSSION TITLE
Pnp Framework/ Core Multi Tenant Authentication, non interactive.

### DISCUSSION THREAD

**Discussion by geedsen** [pnp/pnpframework#565] [Q&A]
I have spent the last few days trying to get our application (the sharepoint part)  upgraded to modern authentication. And I failed...
I have tried so many things without any luck so far. 
We used to use just username and password. But now we want to go through by authenticating a registered application from AD. This is a multi-tenant application and only uses Application Permissions.  I have added Sites.Read.All to these permissions and had one tenant give consent to this. We are also using this application to access MS Graph and that is no problem at all. 
I have tried this by using the AuthenticationManager and also by using the IContextFactory  /PnpCoreAuthentication.  We do not use certificates for authentication, just ClientId and ClientSecret. The sample for Multi Tenant uses an interactive login, which I cannot use. Any help/hints with this would be appreciated.

**Comment by stevebeauge** [comment]
Based on my experience, only certificate authentication is supported on ACS endpoints
